### DISCUSSION TITLE
Has Anyone Tried Using More Data

### DISCUSSION THREAD

**Discussion by rpassaro** [kyleskom/NBA-Machine-Learning-Sports-Betting#394] [Ideas]
I might not be considering something, but has anyone tried using data from 2000, let's say, or even 1980 maybe, instead of 2007? Wouldn't it likely be much more accurate?

**Comment by nealmick** [comment]
The game changes overtime and the further back you go the more fundamental differences there are.  To some degree its all the same game but back in 1980 there were different play styles not as many 3 point shots for example.   I think one would need to capture more data about each game instead of looking at more games.  For example create averages of the last 5, 10, 20 games to capture more momentum.  

Overall any sports analytics is similar to the sport itself, in that people have been competing to analyze or model events for a long time.  People have been studying and improving models for many years for just about anything that you could possibly trade or make money with.  This is not a fresh or untested market.  It's actually fairly saturated and highly competitive.  It just docent seem that way because no one wins and everyone hides their losses.  Its a tarpit, it looks like fresh drinking water, it glimmers in the sun, no one is around, its a safe place to get a drink of water.  But when you step in you get stuck and cant get out.  as you sink, you realize all the people beneath the surface that drank before you....  Here is y combinator talking about tar pits! 
https://www.youtube.com/watch?v=GMIawSAygO4
